pkgs/tools/filesystems/ceph-csi/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, lib, go, ceph, fetchFromGitHub }:
+
+stdenv.mkDerivation rec{
```

Comment:
Please use buildGoModule.

Reply:
At the time of submitting the PR, the go.mod/sum of the project seemed broken. I already tried using buildGoModule and gomod2nix but both failed to build the project. Since the Makefiles just worked out of the box, I didn't want to spend more time on debugging the project gomod.

Reply:
> the go.mod/sum of the project seemed broken

Broken in what way?

Reply:
```suggestion
stdenv.mkDerivation rec {
```